### PR TITLE
Pre-release feedback. Part 4.

### PR DESCRIPTION
**Pull request by enykeev**

### CODE REVIEW

**Comment by dzimine** [comment]
@enykeev please go ahead merge it ASAP and I'll take it for a test. 
@Kami may review it later.

File: modules/st2-menu/template.html [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
   <span class="st2-menu__server-current">
     <i class="st2-icon__user"></i>
-    {{ user || 'Anonymous' }}
+    {{ user || 'Stanley' }}@{{ server.name }}
   </span>
   <div class="st2-menu__server-list">
     <div class="st2-menu__server-current">
       <i class="st2-icon__user"></i>
-      {{ user || 'Anonymous' }}
+      {{ user || 'Stanley' }}@{{ server.name }}
```
Comment (Kami) [inline modules/st2-menu/template.html]: Hm, wait, shouldn't we show an actual username here?

Comment (Kami) [inline modules/st2-menu/template.html]: Ah, nvm, it's `user || ...` It's fine :)

File: modules/st2-report/template.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<a target="_blank" href="https://github.com/stackstorm/st2web/issues">Report a bug</a>
```
Comment (Kami) [inline modules/st2-report/template.html]: :+1:

**Pull request merged**